test/jasmine/assets/check_ticks.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var d3 = require('d3');
+
+module.exports = function checkTicks(axLetter, vals, msg) {
```

Comment:
I've been trying to centralized common custom assertions like this one in:

https://github.com/plotly/plotly.js/blob/master/test/jasmine/assets/custom_assertions.js

I'll let decide whether separate files or a common module is better for stuff like this.

Reply:
oh that's nice - yeah, I'll put `checkTicks` in there.

Reply:
https://github.com/plotly/plotly.js/pull/1999/commits/e4227aad6073d4119cea1d67c3bac2a162701581